Files/Views/Pages/Settings.xaml

```diff
@@ -118,13 +119,13 @@
                 <Rectangle
                     x:Name="Divider"
                     Width=".5"
-                    Margin="0,20,0,20"
+                    Margin="0,0,0,0"
                     HorizontalAlignment="Left"
                     VerticalAlignment="Stretch"
                     Fill="{ThemeResource AppBarItemForegroundThemeBrush}"
-                    Opacity=".5"
+                    Opacity=".35"
                     Visibility="Collapsed" />
-                <Frame x:Name="SettingsContentFrame" Margin=".5,0" />
+                <Frame x:Name="SettingsContentFrame" Padding="14,0,0,0" />
```

Comment:
Will this cause issues with the content getting cut off if you make the window size smaller?

Reply:
Not in my testing.

Reply:
Can you try resizing the window to be the smallest size before the nav menu moves to the top view. One you do that, check on every settings page that nothing gets cut off.